vendor/modules.txt

```diff
@@ -1650,3 +1650,4 @@ tags.cncf.io/container-device-interface/pkg/parser
 # tags.cncf.io/container-device-interface/specs-go v0.8.0
 ## explicit; go 1.19
 tags.cncf.io/container-device-interface/specs-go
+# github.com/containers/image/v5 => github.com/kwilczynski/image/v5 v5.32.3-0.20241128085047-0c20855557c9
```

Comment:
is this really meant to point to an unreleased version?

Reply:
See:

- https://github.com/cri-o/cri-o/pull/8838#discussion_r1882354878